database_admin/migrations/025_packages.up.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS package_name
+(
+    id   INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+    name TEXT                                 NOT NULL CHECK ( NOT empty(name)) UNIQUE
```

Comment:
Some ` NOT empty(name)`s in this file have suffix space, some of them not, so let's unify it, please.